torchvision/csrc/ROIAlign.h

```diff
@@ -1,14 +1,7 @@
 #pragma once
 
-#include "cpu/vision_cpu.h"
```

Comment:
So, those includes might not be necessary for now given that this is a header file, but if we were to convert this to a C++ file it would be needed.

To be kept in mind: we might want to revisit / refactor the way the C++ files are layed down in torchvision so that it follows best practice

Reply:
Sounds good I'll restore it.